### PR TITLE
Migrate `Configu init` command to `@configu/cli`

### LINKED ISSUE #621: Migrate `Configu Init` command to `@configu/cli`
### Suggestion

Return the previous `Configu Init` command

### Motivation

Until the `Configu Init` command is reproductized by @rannn505, we should have the previous init command for the CLI feature completeness.

### Context

Requested by @pelegpor.

### PR DESCRIPTION
**Pull request by RichardAkman**
Closes #621

Notes:
- All references and related logic to the old `type` property was removed from the migrated command
- Migrated some logic from the old `@configu/lib` that are necessary for the command
- The schema generated is adapted to the new version of `ConfigSchema` that's expected by `@configu/common`. The template is commented out for now until the property is returned by `@configu/sdk`
- Fixes an issue in `base.ts` where the `Option` parameter wasn't imported

### CODE REVIEW

File: packages/cli/src/helpers/configExtractor.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import Dotenv from 'dotenv';
+import _ from 'lodash';
+import { Cfgu, Config } from '@configu/sdk';
+
+type ConfigFormat =
```
Comment (davidmeirlevy) [inline packages/cli/src/helpers/configExtractor.ts:5]: that's the old implentation

File: packages/cli/src/helpers/configExtractor.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  'JSON',
+  'Dotenv',
+]);
+
+const FILE_CONTENT_FORMAT_ERROR =
+  'file content is not in a valid format. for more info on supported formats read here: https://configu.com/docs';
+const NOT_FLAT_JSON_ERROR = 'only flat .json files are supported. read more here: https://configu.com/docs';
+
+const CONFIG_EXTRACTORS: Record<ConfigExtractorFormat, ExtractorFunction> = {
+  Dotenv: ({ content }) => Dotenv.parse(content),
```
Comment (davidmeirlevy) [inline packages/cli/src/helpers/configExtractor.ts:41]: we should use the new integrations

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]

File: packages/cli/src/helpers/configExtractor.ts (hunk 2 above)
Comment (RichardAkman) [reply packages/cli/src/helpers/configExtractor.ts:41]: The integrations are not relevant, they are for stringification while this logic is for parsing, two different purposes.

File: packages/cli/src/helpers/configExtractor.ts (hunk 1 above)
Comment (RichardAkman) [reply packages/cli/src/helpers/configExtractor.ts:5]: The old implementation is what @pelegpor asked for. Once Ran is back, the `init` command will be reworked and this code will be changed accordingly.

File: packages/cli/src/helpers/configExtractor.ts (hunk 2 above)
Comment (davidmeirlevy) [reply packages/cli/src/helpers/configExtractor.ts:41]: Maybe we'll need another task to convert the usage here to the adaptive expressions, but that's another task.

**Review by davidmeirlevy — APPROVED** [review]

**Pull request merged**